install.sh

```diff
@@ -9,11 +9,14 @@ abort() {
 	exit 1
 }
 
-SUDO=""
-if [ ! "$UID" -eq 0 ]; then
-	sudo -l mkdir &>/dev/null || abort "This installer requires root or sudo."
-	SUDO="sudo "
-fi
+sudo() {
+	if [ "$UID" -eq 0 ]; then
+ 		command "${@}"
+		return
+	fi
+
+ 	/usr/bin/sudo "${@}"
```

Comment:
We're making an assumption here that sudo is in /usr/bin/sudo, but it could be in a different place.

Reply:
Resolved 🍏